inst/shiny/modules/parameter_datasets.R

```diff
@@ -0,0 +1,75 @@
+parameter_datasets_ui <- function(id) {
+  ns <- NS(id)
+  navset_pill(
+    nav_panel("PP", DTOutput(ns("pp_dataset"))),
+    nav_panel("ADPP", DTOutput(ns("adpp_dataset")))
+  )
+}
+
+parameter_datasets_server <- function(id, res_nca) {
+  moduleServer(id, function(input, output, session) {
+    CDISC <- reactive({
+      req(res_nca())
+      export_cdisc(res_nca())
+    })
+
+    output$pp_dataset <- DT::renderDataTable({
+      DT::datatable(
+        data = CDISC()$pp,
+        rownames = FALSE,
+        extensions = c("FixedHeader", "Buttons"),
+        options = list(
+          scrollX = TRUE,
+          scrollY = TRUE,
```

Comment:
suggestion: allow user to copy/download all rows with a whole display of the dataset

```suggestion
          scrollY = TRUE,
          pageLength = -1,
```